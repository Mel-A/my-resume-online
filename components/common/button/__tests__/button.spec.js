import { shallow } from 'enzyme'
import Button from '..'

describe('Button', () => {
  it('button renders correctly', () => {
    const dummyText = 'text'
    const button = shallow(<Button text="text">{dummyText}</Button>)
    expect(button).toMatchSnapshot()
  })
})
