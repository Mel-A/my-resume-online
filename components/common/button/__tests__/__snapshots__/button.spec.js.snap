// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button button renders theme correctly 1`] = `ReactWrapper {}`;
